packages/patternfly-4/react-core/src/components/Radio/Radio.tsx

```diff
@@ -1,14 +1,21 @@
-import * as React from 'react';
-import styles from '@patternfly/react-styles/css/components/Radio/radio';
-import { css, getModifier } from '@patternfly/react-styles';
-import { Omit } from '../../helpers/typeUtils';
+import * as React from "react";
+import styles from "@patternfly/react-styles/css/components/Radio/radio";
+import { css, getModifier } from "@patternfly/react-styles";
+import { Omit } from "../../helpers/typeUtils";
 
 export interface RadioProps
-  extends Omit<React.HTMLProps<HTMLInputElement>, 'disabled' | 'label' | 'onChange' | 'type'> {
+  extends Omit<
+    React.HTMLProps<HTMLInputElement>,
+    "disabled" | "label" | "onChange" | "type"
+  > {
   /** Additional classes added to the radio. */
   className?: string;
   /** Id of the radio. */
   id: string;
+  /** Flag to show if the radio label is wrapped on small screen. */
+  isWrapped?: boolean;
```

Comment:
Maybe we can be little more descriptive with prop names here? e.g. isLabelWrapped and is isLabelBeforeButton.

Reply:
I like that idea